.ci/lint_r_code.R

```diff
@@ -33,6 +33,7 @@ LINTERS_TO_USE <- list(
     , "assignment"           = lintr::assignment_linter()
     , "braces"               = lintr::brace_linter()
     , "commas"               = lintr::commas_linter()
+    , "class_equals"         = lintr::class_equals_linter()
```

Comment:
Use alphabetical order please.
```suggestion
    , "class_equals"         = lintr::class_equals_linter()
    , "commas"               = lintr::commas_linter()
```

Reply:
Should be fixed!